### PR TITLE
New values for OKTA Rollout

### PR DESCRIPTION
**Pull request by supriyaaddagada**
This PR is to add new values for OKTA rollout and make sure to not have the impact on existing setup.

### CODE REVIEW

File: operations/app/terraform/vars/prod/locals.tf [hunk 1]
```diff
@@ -2,15 +2,18 @@ locals {
...
-    storage_queue_name    = ["process"]
+    environment          = local.environment
+    location             = "eastus"
+    is_metabase_env      = true
+    resource_group_name  = "prime-data-hub-${local.environment}"
+    resource_prefix      = "pdh${local.environment}"
+    okta_redirect_url    = "https://prime.cdc.gov/download"
+    okta_base_url        = "hhs-prime.okta.com"
+    OKTA_scope           = "simple_report_dev"
+    RS_okta_base_url     = "https://reportstream.oktapreview.com"
```
Comment (arnejduranovic) [inline operations/app/terraform/vars/prod/locals.tf:13]: Why does `RS_okta_base_url` have https in it but the original doesn't?

File: operations/app/terraform/modules/function_app/locals.tf [hunk 2]
```diff
@@ -12,6 +12,9 @@ locals {
     "OKTA_redirect" = var.okta_redirect_url
     "OKTA_authkey"  = var.OKTA_authKey
     "OKTA_ClientId" = var.OKTA_clientId
+    "RS_OKTA_redirect" = var.RS_okta_redirect_url
```
Comment (arnejduranovic) [inline operations/app/terraform/modules/function_app/locals.tf:15]: missing `"RS_OKTA_baseUrl"  = var.RS_okta_base_url`?

**Review by arnejduranovic — CHANGES_REQUESTED** [review]
Before merging, let's merge this branch to the branch containing app code and deploy that to a demo env to test these changes. @jeremy-page do you see any problems with that? Will these TF changes get uploaded to the demo env if we do it that way?

File: operations/app/terraform/vars/prod/locals.tf (hunk 1 above)
Comment (jpandersen87) [reply operations/app/terraform/vars/prod/locals.tf:13]: `RS_okta_base_url` definitely needs to be formatted the same (without `https://`)

Comment (supriyaaddagada) [reply operations/app/terraform/vars/prod/locals.tf]: Made this update in the latest commit

File: operations/app/terraform/modules/function_app/locals.tf (hunk 2 above)
Comment (supriyaaddagada) [reply operations/app/terraform/modules/function_app/locals.tf:16]: Added RS_OKTA_baseUrl in the latest commit

**Review by arnejduranovic — APPROVED** [review]

File: operations/app/terraform/vars/demo/locals.tf [hunk 3]
```diff
@@ -3,16 +3,19 @@ locals {
...
+    random_id            = local.random_id
+    resource_group_name  = "prime-data-hub-${var.environment}"
+    resource_prefix      = "pdh${var.environment}"
+    okta_redirect_url    = "https://${var.environment}.prime.cdc.gov/download"
+    okta_base_url        = "hhs-prime.oktapreview.com"
+    OKTA_scope           = "simple_report_dev"
     sftp_container_module = false
+    RS_okta_base_url     = "reportstream.oktapreview.com"
+    RS_okta_redirect_url = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
+    RS_OKTA_scope        = "simple_report_dev"
```
Comment (jpandersen87) [inline operations/app/terraform/vars/demo/locals.tf:18]: `RS_OKTA_scope` is `"reportstream_dev"`

File: operations/app/terraform/vars/prod/locals.tf [hunk 4]
```diff
@@ -2,15 +2,18 @@ locals {
...
-    storage_queue_name    = ["process"]
+    environment          = local.environment
+    location             = "eastus"
+    is_metabase_env      = true
+    resource_group_name  = "prime-data-hub-${local.environment}"
+    resource_prefix      = "pdh${local.environment}"
+    okta_redirect_url    = "https://prime.cdc.gov/download"
+    okta_base_url        = "hhs-prime.okta.com"
+    OKTA_scope           = "simple_report_dev"
+    RS_okta_base_url     = "reportstream.oktapreview.com"
```
Comment (jpandersen87) [inline operations/app/terraform/vars/prod/locals.tf:13]: `RS_okta_base_url` is `"reportstream.okta.com"`

File: operations/app/terraform/vars/prod/locals.tf [hunk 5]
```diff
@@ -2,15 +2,18 @@ locals {
...
+    RS_okta_redirect_url = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
+    RS_OKTA_scope        = "simple_report_dev"
```
Comment (jpandersen87) [inline operations/app/terraform/vars/prod/locals.tf:15]: `RS_OKTA_scope` is `"reportstream_prod"`

File: operations/app/terraform/vars/staging/locals.tf [hunk 6]
```diff
@@ -2,15 +2,18 @@ locals {
...
+    location             = "eastus"
+    is_metabase_env      = true
+    resource_group_name  = "prime-data-hub-${local.environment}"
+    resource_prefix      = "pdh${local.environment}"
+    okta_redirect_url    = "https://${local.environment}.prime.cdc.gov/download"
+    okta_base_url        = "hhs-prime.oktapreview.com"
+    OKTA_scope           = "simple_report_dev"
+    RS_okta_redirect_url = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
+    RS_okta_base_url     = "reportstream.oktapreview.com"
+    RS_OKTA_scope        = "simple_report_dev"
```
Comment (jpandersen87) [inline operations/app/terraform/vars/staging/locals.tf:15]: `RS_OKTA_scope` is `"reportstream_dev"`

File: operations/app/terraform/vars/test/locals.tf [hunk 7]
```diff
@@ -10,6 +10,9 @@ locals {
     okta_redirect_url     = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
     okta_base_url         = "hhs-prime.oktapreview.com"
     OKTA_scope            = "simple_report_dev"
+    RS_okta_base_url      = "reportstream.oktapreview.com"
+    RS_okta_redirect_url  = "https://prime-data-hub-XXXXXXX.azurefd.net/download"
+    RS_OKTA_scope         = "simple_report_dev"
```
Comment (jpandersen87) [inline operations/app/terraform/vars/test/locals.tf:15]: `RS_OKTA_scope` is `"reportstream_dev"`

**Review by jpandersen87 — CHANGES_REQUESTED** [review]

File: operations/app/terraform/vars/test/locals.tf (hunk 7 above)
Comment (supriyaaddagada) [reply operations/app/terraform/vars/test/locals.tf]: RS_OKTA_scope has been updated in the latest commit

File: operations/app/terraform/vars/staging/locals.tf (hunk 6 above)
Comment (supriyaaddagada) [reply operations/app/terraform/vars/staging/locals.tf]: RS_OKTA_scope has been updated in the latest commit

File: operations/app/terraform/vars/prod/locals.tf (hunk 5 above)
Comment (supriyaaddagada) [reply operations/app/terraform/vars/prod/locals.tf]: RS_OKTA_scope has been updated in the latest commit

File: operations/app/terraform/vars/prod/locals.tf (hunk 4 above)
Comment (supriyaaddagada) [reply operations/app/terraform/vars/prod/locals.tf]: RS_okta_base_url has been updated in the latest commit

File: operations/app/terraform/vars/demo/locals.tf (hunk 3 above)
Comment (supriyaaddagada) [reply operations/app/terraform/vars/demo/locals.tf]: RS_OKTA_scope has been updated in the latest commit

**Review by jpandersen87 — APPROVED** [review]

**Review by jeremy-page — APPROVED** [review]
Excellent, will apply to Prod once we merge to master.

**Pull request merged**